core/modules/utils/utils.js

```diff
@@ -799,5 +799,22 @@ exports.strEndsWith = function(str,ending,position) {
 		return lastIndex !== -1 && lastIndex === position;
 	}
 };
+	
+/**
+ * @function setTags
+ * @description Helper function to override the tags of a tiddler
+ * If the tiddler does not exist it will be created
+ * @param  {String} title Tiddler you want to override its' tags
+ * @param  {[String]} tags  List of tags that will be the new tiddler tags
```

Comment:
There is no check in the code, if the parameter is valid

Reply:
IMO it should look more like: FieldManglerWidget.prototype.handleAddTagEvent

Reply:
I suggested, that your function should implement several checks, if the parameters are valid. Similar to: https://github.com/Jermolene/TiddlyWiki5/blob/master/core/modules/widgets/fieldmangler.js#L128

Reply:
>Or are you suggesting to use that method to accomplish this task ?

no